tutorials/tutorial2.ipynb

```diff
@@ -12,7 +12,7 @@
     "\n",
     "We selected 2400 NK cells from this dataset - this is our ‘clean’, high-coverage dataset. We then randomly sampled 50 of these 2400 NK cells. Here's what the ATAC-seq signal from 50 cells and 2400 cells looks like, for a region on chromosome 10:\n",
     "\n",
-    "![subsampled_NK_cells](../docs/tutorials/NK.2400.50.png)\n",
+    "![subsampled_NK_cells](NK.2400.50.png)\n",
     "\n",
     "Compared to the 'clean' signal from 2400 cells, the aggregated ATAC-seq profile of these 50 cells is noisy. Because the signal is noisy, peak calls calculated by MACS2 on this data (shown as red bars below the signal tracks) are also inaccurate. The AUPRC of peak calling by MACS2 on the noisy data is only 0.20.\n",
     "\n",
```

Comment:
Above line 21, can we link to the section of the documentation where we talk about caveats for the use of pre-trained models?

Reply:
done.